ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -329,9 +360,12 @@ runDataDiffusion tracers
           case daLocalAddress of
                Left _ -> pure () -- If a socket was provided it should be ready to accept
                Right path -> do
+                 traceWith dtDiffusionInitializationTracer . ConfiguringLocalSocket path =<< localSocketFileDescriptor sd
```

Comment:
I never liked `=<<`, but it actually make sense here.  Maybe I should change my mind about it.

Reply:
Just break the line in two, it's too long, e.g.
```
traceWith dtDiffusionInitializationTracer . ConfiguringLocalSocket path
  =<< localSocketFileDescriptor sd
```